tests/percy/multilangphotographer.js

```diff
@@ -0,0 +1,170 @@
+/**
+ * @typedef {import('./pagenavigator.js')} PageNavigator
+ * @typedef {import('./Camera.js')} Camera
+ */
+
+/**
+ * Responsible for determining which snapshots to take for specific locale
+ */
+class MultilangPhotographer {
```

Comment:
Would it make sense to consolidate down to a single `Photographer` class? I guess it'd effectively be what we have here, since it's more general. What do you think @cea2aj ?

Reply:
Right now `MultilangPhotographer` covers majority of the snapshot tests from `Photographer`(beside directanswers document search) - I'm not sure if we actually want all those snapshots, and if we want to separate it this way for when we may want to include for specific test scenarios only for single/multilang variants in the future.

Reply:
Currently the responsibility of the `Photographer` is to define the test suite itself. I suggested that we make a `MultilangPhotographer` as a way to break up the test suite so that we don't have to have all of the snapshots in a single file. I have another idea for allowing us to split up the tests into multiple files. I think this change would be out of scope of this PR, but i'll include in here for reference:

Rather than having the photographer define the test suite, the test suite could instead be defined by files in particular folder (similar to how jest and test cafe work). The files would contain exported functions which define the snapshots to take, and they would each be passed a page navigator and a camera. For multilang we could pass an additional object which contains language-specific strings for use in the tests. At this point, the photographer might not be necessary anymore